dataprofiler/profilers/utils.py

```diff
@@ -7,6 +7,7 @@
 import psutil
 import numpy as np
 import multiprocessing as mp
+from dataprofiler import settings
```

Comment:
this needs to be spaced from the top 1st block of imports. For imports the structure is:
1. built-ins
2. 3rd party
3. Own library